client/src/views/Curse.view.tsx

```diff
@@ -0,0 +1,134 @@
+        }
+    },
+    {
+        numeCurse: "1735masculin",
+        interfata:{
+            cursa: "Masculin - 17-35ani",
+            oraStart: "13 : 00",
+            distanta: "6 KM (3 X 2 KM)",
+            donatieMinima: "40 RON",
+            urlHarta: "https://www.google.com/maps/embed?pb=!1m52!1m12!1m3!1d1359.0739952763695!2d26.051536155519436!3d44.4381565085272!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m37!3e2!4m3!3m2!1d44.4372435!2d26.047853099999998!4m3!3m2!1d44.4392806!2d26.047387!4m3!3m2!1d44.4388363!2d26.0495043!4m3!3m2!1d44.438002399999995!2d26.051835699999998!4m3!3m2!1d44.440619999999996!2d26.0526777!4m3!3m2!1d44.4378836!2d26.054248899999997!4m3!3m2!1d44.4377256!2d26.051095999999998!4m3!3m2!1d44.4380264!2d26.0493903!4m3!3m2!1d44.4372292!2d26.0480509!5e0!3m2!1sro!2sro!4v1679944496994!5m2!1sro!2sro",
+        }
+    },
+    {
+        numeCurse: "35masculin",
+        interfata:{
+            cursa: "Masculin 35+ ani",
+            oraStart: "15 : 00",
+            distanta: " 4 KM (2 x 2 KM)",
+            donatieMinima: "40 RON",
+            urlHarta: "https://www.google.com/maps/embed?pb=!1m52!1m12!1m3!1d1359.0739952763695!2d26.051536155519436!3d44.4381565085272!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m37!3e2!4m3!3m2!1d44.4372435!2d26.047853099999998!4m3!3m2!1d44.4392806!2d26.047387!4m3!3m2!1d44.4388363!2d26.0495043!4m3!3m2!1d44.438002399999995!2d26.051835699999998!4m3!3m2!1d44.440619999999996!2d26.0526777!4m3!3m2!1d44.4378836!2d26.054248899999997!4m3!3m2!1d44.4377256!2d26.051095999999998!4m3!3m2!1d44.4380264!2d26.0493903!4m3!3m2!1d44.4372292!2d26.0480509!5e0!3m2!1sro!2sro!4v1679944496994!5m2!1sro!2sro",
+        }
+    },
+    {
+        numeCurse: "35feminin",
+        interfata:{
+            cursa: "Feminin 35+ ani",
+            oraStart: "14 : 10",
+            distanta: " 2 KM ",
+            donatieMinima: "40 RON",
+            urlHarta: "https://www.google.com/maps/embed?pb=!1m52!1m12!1m3!1d1359.0739952763695!2d26.051536155519436!3d44.4381565085272!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!4m37!3e2!4m3!3m2!1d44.4372435!2d26.047853099999998!4m3!3m2!1d44.4392806!2d26.047387!4m3!3m2!1d44.4388363!2d26.0495043!4m3!3m2!1d44.438002399999995!2d26.051835699999998!4m3!3m2!1d44.440619999999996!2d26.0526777!4m3!3m2!1d44.4378836!2d26.054248899999997!4m3!3m2!1d44.4377256!2d26.051095999999998!4m3!3m2!1d44.4380264!2d26.0493903!4m3!3m2!1d44.4372292!2d26.0480509!5e0!3m2!1sro!2sro!4v1679944496994!5m2!1sro!2sro",
+        }
+    },
+];
+
+export function getCurse(): string[]{ 
```

Comment:
Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.